dev-support/mini-submarine/submarine/run_submarine_mnist_tony.sh

```diff
@@ -17,8 +17,10 @@
 
 # Below are configurable variables, please adapt base on your local environment.
 # Version of submarine jar
-SUBMARINE_VERSION=0.3.0-SNAPSHOT
-
+SUBMARINE_VERSION=$SUBMARINE_VER
```

Comment:
SUBMARINE_VERSION =${SUBMARINE_VER:-"0.3.0-SNAPSHOT"}
This way, So you can set default values.